### DISCUSSION TITLE
I've build a website that's already dark but Dark Reader doesn't see it, how can I makde it work by default?

### DISCUSSION THREAD

**Discussion by Juraj-Masiar** [darkreader/darkreader#11050] [Q&A] [answered]
Hello,
I'm author of:
https://group-speed-dial.fastaddons.com/
And Dark Reader is not detecting it as dark. I don't want to report it as bad detection, I want to fix my page so that it's detected correctly.
I'm using standard Angular Material dark theme so really nothing special and I was expecting it to work automatically.

**Comment by MikaelFangel** [comment, marked as answer]
It says what to do in [CONTRIBUTING.md](https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md#adding-a-website-that-is-already-dark) (:

**Reply by Juraj-Masiar** [reply]
Great!
But you've linked wrong paragraph, the one above looks like what I need:
https://github.com/darkreader/darkreader/blob/main/CONTRIBUTING.md#disabling-dark-reader-on-your-site
Thanks!
